config/tsconfig.build.json

```diff
@@ -4,9 +4,10 @@
     "allowJs": true
   },
   "exclude": [
-    "tests/js",
-    "fixtures/js-stubs",
+    "../tests/js",
+    "../fixtures/js-stubs",
     "../static/app/**/*.spec.*",
-    "../static/app/**/*.benchmark.ts"
+    "../static/app/**/*.benchmark.ts",
+    "../static/app/views/settings/project/server-side-sampling/testUtils.tsx"
```

Comment:
So `tsconfig.build.json` is now only called when building assets from `setup.py` -- intent is to have it only type application-code and not test-code. Now that tests are co-located, test utility files do not have a pattern right now that we can catch and exclude.

any ideas here? cc @scttcper @evanpurkhiser

Reply:
I'm gonna go ahead and merge this, I can follow-up with this if needed